lang/cs/Org.Apache.REEF.Tests/Org.Apache.REEF.Tests.csproj

```diff
@@ -130,11 +130,6 @@ under the License.
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <Content Include="bin\reef-bridge-java-0.11.0-incubating-SNAPSHOT-shaded.jar">
```

Comment:
Is there a way to refer to this by the current version instead of hardcoding it?